db/helpers/common.js

```diff
@@ -143,7 +143,6 @@ function prepareToPublish(inst, changedKeys, ignoreAttributes) {
   for (let i = 0; i < changedKeys.length; i++) {
     if (!ignoreSet.has(changedKeys[i])) {
       return {
-        old: inst._previousDataValues,
         new: inst.get(),
```

Comment:
@iamigo @annyhe we will need the old values to do some filtering in the realtime module. Can we remove the old values in /realtime/socketIOEmitter.js at line 30, just before the object is emitted to the client ?